components/Wallet/Send.tsx

```diff
@@ -38,12 +42,22 @@ export const Send = ({ walletProviderOpts, pendingMsgContext }: SendProps) => {
   const [txState, setTxState] = useState<TxState>(TxState.FillingForm)
   const [txFee, setTxFee] = useState<FilecoinNumber | null>(null)
 
+  const toAddrFormatted = useMemo<string>(() => {
+    if (isToAddressValid) {
+      return isEthAddress(toAddress)
+        ? delegatedFromEthAddress(toAddress, coinType)
+        : toAddress
+    }
+
+    return ''
+  }, [coinType, toAddress, isToAddressValid])
```

Comment:
I wouldn't care for setting it to an empty string if the original `toAddress` is invalid, makes for a simpler method. But either is fine.

```javascript
const toAddrFormatted = useMemo<string>(
  () => isToAddressValid && isEthAddress(toAddress)
    ? delegatedFromEthAddress(toAddress, coinType)
    : toAddress,
  [coinType, toAddress, isToAddressValid]
)
```